shopware6/deploy-shopware6.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Deployer;
+
+require 'recipe/common.php';
+require 'recipe/rsync.php';
+require 'recipe/cachetool.php';
+
+set('console', 'bin/console');
+set('shopware_build_path', '/tmp/build');
+set('build_script', 'bin/build-js.sh');
+set('rsync', [
+    'exclude'      => [
+        'files/',
+        'var/cache/',
+        'var/log/',
+        '.env',
+        '.git',
+        'deploy.php',
+    ],
+    'exclude-file' => false,
+    'include'      => [],
+    'include-file' => false,
+    'filter'       => [],
+    'filter-file'  => false,
+    'filter-perdir'=> false,
+    'flags' => 'rzE',
+    'options' => [
+        'delete',
+        'links',
+    ],
+    'timeout'      => 3600,
+]);
+
+task('shopware:build:prepare', function () {
+    run('mkdir -p {{shopware_build_path}}');
+})->setPrivate()->local();
+
+task('shopware:filesystem:deploy', function () {
+    run("cp -r {{source_directory}}/* {{shopware_build_path}}/");
+    run("cp -r {{source_directory}}/.* {{shopware_build_path}}/");
+})->setPrivate()->local();
+
+//region install/update
+task('shopware6:install:download', function() {
+    run('curl -sL {{shopware_install_url}} -o {{shopware_build_path}}/download.zip');
+    run('cd {{shopware_build_path}} && unzip -qq download.zip && rm -rf download.zip');
```

Comment:
```suggestion
    run('cd {{shopware_build_path}} && unzip -qqo download.zip && rm -rf download.zip');
```

Reply:
Ich würde eher `-n` vorschlagen. Wenn eine "Core"-Datei im Repo ist sollte die auch dort verbleiben, oder? Ich denke an sowas wie die composer.json und composer.lock, in der wir weitere Bibliotheken/Plugins einbinden könnten.

Reply:
Stimmt. War verwirrt, weil das in einem temporären Verzeichnis passiert. Könnte dann aber beim Update in `shopware6:update` noch interessant werden...